requests/add-collier-and-ninja-static-output.yaml

```diff
@@ -0,0 +1,8 @@
+action: add_feedstock_output
+feedstock_to_output_mapping:
+  - collier:
+    - "collier"
+    - "collier-static"
+  - ninja-hep-ph:
+    - "ninja-hep-ph"
+    - "ninja-hep-ph-static"
```

Comment:
I believe that I'm supposed to list _all_ allowed outputs, not just those I want to add, so listing the shared library `collier` and `ninja-hep-ph` outputs here too.